### ISSUE TITLE
Option to remove year from title or change title

### ISSUE THREAD

**Issue by Newjorciks** [morpheus65535/bazarr#323]
Addic7ed.com search doesn't seem to work with some shows containing year.
At least 'The Grand Tour' doesn't work. Sonarr has 'The Grand Tour' listed as "The Grand Tour (2016)" which bazarr then uses to search for the subtitles, but on addic7ed.com "The Grand Tour (2016)" returns nothing, while "The Grand Tour" does.

I tried out manually changing the title to "The Grand Tour" in the bazarr database and it fetched the subtitles from addic7ed properly. Can't say outright changing the title would work correctly on other subtitle sites though.

An option to remove the year from the title before search would be great.
Another option could be to have a custom name you can set for the show and then enable/disable the usage of custom names for specific subtitle sites.

**Comment by morpheus65535** [comment]
Manually editing the DB is not a good idea and your change will be overwritten anyway. Please try dev branch that include a fix for that. Thanks.

**Comment by Newjorciks** [comment]
I am well aware of how bad directly editing can be and I knew it would be overwritten when synced with sonarr, but I mainly did it for testing to see if that truly is the issue and didn't care too much if I corrupted the DB or bazarr as whole as I can just reinstall it.
I did the DB editing with the docker instance shut down and I knew changing a value such as a simple "title" shouldn't really have broken anything anyways(I am a dev myself).

Tested the dev branch and it downloaded the subtitles for "The Grand Tour (2016)" from addic7ed, so working as expected. I guess this can be closed or maybe close it when this is pushed to master, so that anyone that sees this knows it's not in master already.

Thank you for your work on bazarr!
